_posts/2020-01-01-Resurrecting-Sigmoid.markdown

```diff
@@ -0,0 +1,521 @@
+---
+layout: post
+title:  "Resurrecting the sigmoid in deep learning through dynamical isometry: theory and practice"
+date:   2020-01-01 6:00:00 -0400
+categories: neural-nets
+author: "Jeffrey Pennington, Samuel S. Schoenholz, Surya Ganguli"
+blurb: "Curriculum covering the paper Resurrecting the sigmoid in deep learning through dynamical isometry: theory and practice, by Pennington et al"
+feedback: true
+---
+
```

Comment:
nit: Add `# Acknowledgments` header? Not super necessary though so I'll defer to what you guys think